lib/screenplay/util.ex

```diff
@@ -10,4 +10,20 @@ defmodule Screenplay.Util do
     username
     |> String.replace("ActiveDirectory_MBTA\\", "")
   end
+
+  def happening_now?(%{active_period: aps}, now \\ DateTime.utc_now()) do
+    Enum.any?(aps, &in_active_period(&1, now))
+  end
+
+  def in_active_period({nil, end_t}, t) do
+    DateTime.compare(t, end_t) in [:lt, :eq]
+  end
+
+  def in_active_period({start_t, nil}, t) do
+    DateTime.compare(t, start_t) in [:gt, :eq]
+  end
+
+  def in_active_period({start_t, end_t}, t) do
+    DateTime.compare(t, start_t) in [:gt, :eq] && DateTime.compare(t, end_t) in [:lt, :eq]
+  end
```

Comment:
The logic here looks good, but I think adding these to `alert.ex` makes a little more sense.